views/editor/bramble.html

```diff
@@ -37,6 +37,16 @@
             -webkit-animation: loadingpulse 1s infinite linear;
         }
 
+        #spinner-container .loading-message-help-link {
+            animation: none;
+            -webkit-animation: none;
```

Comment:
Always put non-prefixed versions of CSS rules at the end of your list, so that the spec standard one gets used if it's available.